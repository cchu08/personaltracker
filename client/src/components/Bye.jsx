import React from 'react';

const Bye = () =>
  <div>
    <h1>Bye World.</h1>
  </div>;

export default Bye;
