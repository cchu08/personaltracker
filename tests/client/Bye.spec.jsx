import React from 'react';
import { shallow } from 'enzyme';
import Bye from '../../client/src/components/Bye';

describe('Bye component', () => {
  const component = shallow(<Bye />);
  
  it('renders correctly', () => {
    expect(component).toMatchSnapshot();
  });

  it('should have an h1 tag that says Bye', () => {
    expect(component.find('h1').html()).toContain('Bye');
  });
});