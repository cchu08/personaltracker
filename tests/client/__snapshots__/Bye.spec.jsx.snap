// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Bye component renders correctly 1`] = `
<div>
  <h1>
    Bye World.
  </h1>
</div>
`;
